src/wp-admin/edit-form-advanced.php

```diff
@@ -462,12 +462,17 @@
 		)
 	);
 endif;
+
+$connection_lost_message = '<span class="spinner"></span> ' . __( '<strong>Connection lost.</strong> Saving has been disabled until you are reconnected.' ) . '<span class="hide-if-no-sessionstorage">' . __( 'This post is being backed up in your browser, just in case.' ) . '</span>';
+
+wp_admin_notice(
+	$connection_lost_message,
+	array(
+		'id'                 => 'lost-connection-error',
```

Comment:
```suggestion
		'id'                 => 'lost-connection-notice',
```
The original ID was `lost-connection-notice`. Was `lost-connection-error` a mistake or an intentional change?

Reply:
Probably a cut and paste from a previous change; wasn't intentionally changing anything.